### DISCUSSION TITLE
[Question] Adding Information to ng alert email output

### DISCUSSION THREAD

**Discussion by unnuetz** [grafana/grafana#44051] [Alerting]
Do somebody know what exactly I have to add to the email template html file, that this following information on a grafana alert will be added to the email?

![image](https://user-images.githubusercontent.com/25468412/149526922-dd1420d8-b2f2-421c-9924-c235d598195e.png)

I want to have the full "Info" inserted in the Email which will be sent.

I tried to add {{ .Info }} in the .html file but this does not work. (or Value)

The Information i found is:
https://sbcode.net/grafana/email-alert-template/

but this doesnt really helped me.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
